src/transformers/trainer_utils.py

```diff
@@ -46,7 +46,27 @@
     import tensorflow as tf
 
 
-def set_seed(seed: int):
+def seed_worker():
+    worker_seed = torch.initial_seed() % 2**32
+    set_seed(worker_seed)
+
+
+def set_deterministic_seed_for_cuda():
```

Comment:
I'd change the name of this function, as it doesn't set a seed at all. Perhaps `enable_determinism`?